recipes/models.py

```diff
@@ -1,3 +1,75 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
 
-# Create your models here.
+DIFFICULTY_CHOICES=[('easy','Easy'),
+                    ('moderate','Moderate'),
+                    ('difficult','Difficult')
+]
+
+UNIT_CHOICES=[('gms','Grams'),
+              ('numbers','Numbers'),
+              ('teaspoon','Teaspoon'),
+              ('tablespoon','Tablespoon'),
+              ('litres','Litres')
+]
+
+FOOD_TYPE_CHOICE=[
+    ('veg','Vegetarian'),
+    ('vegan','Vegan'),
+    ('non_veg','Non_Vegetarian'),
+]
+
+CUISINE_CHOICES = [
+    ('italian', 'Italian'),
+    ('indian', 'Indian'),
+    ('mexican', 'Mexican'),
+    ('chinese', 'Chinese'),
+    ('japanese', 'Japanese'),
+]
+
+class Recipe(models.Model):
+    title=models.CharField(max_length=200)
+    author=models.ForeignKey(User,on_delete=models.CASCADE,related_name='recipes')
+    image=models.ImageField(upload_to='images/',blank=True,null=True)
```

Comment:
Should allow multiple images for a recipe. You might have to create another model for recipe images.